include/deal.II/base/geometry_info.h

```diff
@@ -3879,8 +3879,8 @@ GeometryInfo<2>::min_cell_refinement_case_for_face_refinement(
   AssertIndexRange(face_no, GeometryInfo<dim>::faces_per_cell);
 
   if (face_refinement_case == RefinementCase<dim>::cut_x)
-    return (face_no / 2) != 0u ? RefinementCase<dim>::cut_x :
-                                 RefinementCase<dim>::cut_y;
+    return (face_no / 2) ? RefinementCase<dim>::cut_x :
+                           RefinementCase<dim>::cut_y;
```

Comment:
IMHO converting `face_no/2` implicitly to a bool isn't better. Do you think so?
`face_no/2 != 0` essentially means `face_no >=2` I would prefer that.